test/k8s-integration/main.go

```diff
@@ -51,7 +51,7 @@ var (
 
 	// Test infrastructure flags
 	boskosResourceType = flag.String("boskos-resource-type", "gce-project", "name of the boskos resource type to reserve")
-	storageClassFile   = flag.String("storageclass-file", "", "name of storageclass yaml file to use for test relative to test/k8s-integration/config")
+	storageClassFile   = flag.String("storageclass-file", "", "name of storageclass yaml file to use for test relative to test/k8s-integration/config. This may be a comma-separated list to test multiple storage classes")
```

Comment:
should we rename it "storageclass-files"?

Reply:
I didn't know if that would break any existing jobs. Is everything on prow etc run through the run-k8s-integration*.sh scripts?

Reply:
it goes through the script, so if we modify the script we should be fine

Reply:
Cool thanks done.